url-shortner/urlshortner/handler.go

```diff
@@ -0,0 +1,79 @@
+package urlshortner
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"gopkg.in/yaml.v2"
+)
+
+func readYAMLFile() map[string]string {
```

Comment:
More or less the same arguments from the previous PR, better to make this function to consume path and also return an error.